src/index.ts

```diff
@@ -17,7 +17,8 @@ export * from './vanilla'
 const isSSR =
   typeof window === 'undefined' ||
   !window.navigator ||
-  /ServerSideRendering|^Deno\//.test(window.navigator.userAgent)
+  /ServerSideRendering|^Deno\//.test(window.navigator.userAgent) ||
+  Object.keys(window.navigator).length === 0
```

Comment:
I'd prefer this instead.
```diff
-   !window.navigator ||
+   !window.navigator?.userAgent ||
```
Or if there's another way to reduce code, it would be nice.

Reply:
But, are we sure if all client browsers has userAgent?? No false positives?